debian/postinst

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+set -e
+
+if [ "$1" = "configure" ]; then
+	mkdir -p /var/lib/vast
+    if ! getent passwd vast >/dev/null; then
+        adduser \
+            --system --group --no-create-home --home /var/lib/vast \
+            vast
+    fi
+    if ! dpkg-statoverride --list /var/lib/vast >/dev/null 2>&1; then
+        dpkg-statoverride \
+            --quiet --update \
+            --add vast vast 0750 /var/lib/vast
+    fi
+    mkdir -p /var/log/vast
+    if ! dpkg-statoverride --list /var/log/vast >/dev/null 2>&1; then
+        dpkg-statoverride \
+            --quiet --update \
+            --add vast vast 0750 /var/log/vast
+    fi
+fi
+
+# Automatically added by dh_installsystemd/13.3.4
+if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ] ; then
+	# This will only remove masks created by d-s-h on package removal.
+	deb-systemd-helper unmask 'vast.service' >/dev/null || true
```

Comment:
If the postinstall script references our systemd service then we should actually install it like we do for rc.d and launchd.